docs/source/how_to/use_oauth2.rst

```diff
@@ -147,9 +147,59 @@ However, if the response is not successful, you’ll receive an error response.
 Using the access token
 ----------------------
 
-.. note::
+Now that you have the access token, you can get the user data by sending a
+``GET`` request to ``/open_auth/get_user_data/`` with the access token in the
+header as follows:
+
+.. code-block:: json
+
+  {
+    "Authorization": "Bearer vqygrcouTuyAYHZLz3rGcZf5FpPd3K"
```

Comment:
Is the token length constant? If it is,
```suggestion
    "Authorization": "Bearer <n-character access token>"
```
would be better imo.

Reply:
I had used the same access token as the one in the example for [Getting the access and refresh token](https://github.com/IMGIITRoorkee/omniport-docs/commit/6c3f25d6b9deef7941ff6184a4385377b73a4214#diff-d3ac9cfb0f44f5b7c9ddd7b687bb893cR132) to keep it consistent.

If this is changed to `<n-character access token>` then that should also be changed right?

Reply:
If it's consistent, it's okay.